### DISCUSSION TITLE
API데이터솔루션(유료) 문의

### DISCUSSION THREAD

**Discussion by githubcnsbs** [commerce-api-naver/commerce-solutions#21] [묻고 답하기] [answered]
커머스API센터 문서 중 API데이터솔루션(유료) API를 솔루션에서 사용할 수 있는 방법이 있을까요.

예시로 
1. 스마트스토어 셀러가 데이터솔루션을 유료 구독 후 다른 개발사 솔루션에서 해당 API(고객데이터, 실시간분석, 마케팅분석 등 API)를 사용 가능한 방법이 있는지 
2. 혹은 솔루션사가 직접 구독 후 사용할 수 있는 방법이 있을지 문의

**Comment by commerce-api-naver** [comment, marked as answer]
안녕하세요. 커머스API 이민석입니다.

'API데이터솔루션'은 유상 제공에 따른 전용 애플리케이션으로 제공됩니다.
이 전용 애플리케이션은 판매자 분들께 제공하는 '내스토어 애플리케이션'과 커머스솔루션 개발사에 제공하는 '솔루션 애플리케이션'이 다른 것과 같이 혼용이 불가능한 별도의 애플리케이션 유형입니다.

### 첫 번째 문의에 대한 답변
> 스마트스토어 셀러가 데이터솔루션을 유료 구독 후 다른 개발사 솔루션에서 해당 API(고객데이터, 실시간분석, 마케팅분석 등 API)를 사용 가능한 방법이 있는지

판매자가 이용중인 'API데이터솔루션'은 대상 판매자에게만 제공하는 전용 애플리케이션입니다.
이를 다른 개발사 솔루션에서 동일한 기능을 제공받는 방식으로 이용할 수는 없으며, 오로지 해당 판매자가 이용하는 목적으로만 제공합니다.

### 두 번째 문의에 대한 답변
> 혹은 솔루션사가 직접 구독 후 사용할 수 있는 방법이 있을지 문의

현재 커머스솔루션 기술 구조상으로 'API데이터솔루션' 기능은 솔루션사가 이용할 수 없습니다.
